geps/gep-3793/index.md

```diff
@@ -0,0 +1,144 @@
+for the Gateway to use.
+
+The root cause of this friction is a difference in perspective: to Chihiro and
+Ian, the Gateway is a first-class thing that they think about regularly, while
+to Ana, it's an implementation detail that she doesn't care about. Neither
+point of view is wrong, but they are in tension with each other.
+
+### Prior Art
+
+This is very much not a new problem: there are many other systems out there
+where being unambiguous is crucial, but where being completely explicit is a
+burden. One of the simplest examples is the humble URL, where the port number
+is not always explicit, but it _is_ always unambiguous. Requiring everyone to
+type `:80` or `:443` at the end of the host portion of every URL wouldn't
+actually help anyone, though allowing it to be specified explicitly when
+needed definitely does help people.
+
+We can also find prior art where the lack of explicitness _is_ a problem: for
+example, the North American telephone system. North American telephone numbers
+are always ten digits long, split into three parts: the three-digit area code,
+the three-digit exchange number, and the four-digit line number. Many years
+ago, you could place a phone call by dialing just the exchange and line
+numbers, if the area code was the same as your own. People found this
+convenient, but over the years it led to massive confusion as area codes
+became full and had to be split. Today, you always have to specify the area
+code when using a phone in North America: it's more digits, but it's less
+operationally complex, and it removes actual problematic ambiguity.
+
+It's rare to find systems that are purely one or the other: in general, the
+trick is to find a usable balance between explicitness and simplicity, while
+managing ambiguity.
+
+[Chihiro]: https://https//gateway-api.sigs.k8s.io/concepts/roles-and-personas/#chihiro
+[Ian]: https://https//gateway-api.sigs.k8s.io/concepts/roles-and-personas/#ian
+
+### The Problems
+
+## API
+
+Most of the API work for this GEP is TBD at this point. The challenge is to
+find a way to allow Ana to use Routes without requiring her to specify the
+Gateway explicitly, while still allowing Chihiro and Ian to retain control
+over the Gateway and its configuration.
+
+An additional concern is CD tools and GitOps workflows: any solution that
+mutates the `spec` of a Kubernetes resource tends to cause problems for these
+tools. While it's possible to tell tools like Argo CD and Flux to ignore
+certain elements of the `spec` that are known to mutate, the `parentRef` is
+definitely _not_ a field where changes should be ignored! This strongly
+implies that the ideal API will not mutate the `spec` of the Route.
+
+### Gateway for Ingress (North/South)
+
+### Gateway For Mesh (East/West)
+
+## Conformance Details
+
+#### Feature Names
+
+The default-gateway feature will be named `HTTPRouteDefaultGateway` and
+`GRPCRouteDefaultGateway`. It is unlikely that an implementation would support
```

Comment:
I suspect that including separate features for separate route types will be a detail that Ana doesn't want to deal with.